package.json

```diff
@@ -25,7 +25,7 @@
     "deploy": "sls deploy",
     "sls-remove": "sls remove",
     "package": "sls package",
-    "serve": "REQUEST_LOGGING_FORMAT=dev LOG_LEVEL=debug STAC_API_URL=http://localhost:3000 ENABLE_TRANSACTIONS_EXTENSION=true nodemon --esm ./src/lambdas/api/local.ts",
+    "serve": "OPENSEARCH_HOST=https://1zdhk1d5mp7vtjqzphkk.us-west-2.aoss.amazonaws.com REQUEST_LOGGING_FORMAT=dev LOG_LEVEL=debug STAC_API_URL=http://localhost:3000 ENABLE_TRANSACTIONS_EXTENSION=true nodemon --esm ./src/lambdas/api/local.ts",
```

Comment:
This feels awkward/bad to have in the build scripts, no? Feels like something that should be part of a `Developing` section in a README, e.g.

> To serve this project, for development, you need to set `OPENSEARCH_HOST` first, e.g.:
>
> ```shell
> OPENSEARCH_HOST=https://your-opensearch-service-name.aws-region.aoss.amazonaws.com npm serve
> ```